### DISCUSSION TITLE
Anyone try to run Qt app on wasmer?

### DISCUSSION THREAD

**Discussion by stereomatchingkiss** [wasmerio/wasmer#2616] [Q&A]
Do anyone try to run Qt app on wasmer? Any tutorials about this? The tutorials I can found are "running Qt app on browser by wasm", but not "Run Qt app on wasmer".

Thanks

**Comment by jcaesar** [comment]
I suspect you won't find a tutorial for this:
Qt for WASM needs some things that are only available in a browser, e.g. a canvas it can do webgl on.
I've had a short look at the Qt-WASM-examples, and they import an entire load of functions like `env._emscripten_glRenderbufferStorage` from JS.
It's not like it would be impossible to re-implement those in Rust (or whatever language you use wasmer from), but certainly not tutorial-level easy.
